lib/registries/addon/my-registrations/styles.scss

```diff
@@ -25,18 +25,20 @@
         margin-left: 15px;
     }
 
-    :global(a.nav-link) {
+    .NavItem {
         border: none;
         color: #fff;
+        background: none;
+        padding: 10px 15px;
 
         &:hover {
             border: none;
             color: $color-text-gray;
-            background-color: #e4e4e4;    
+            background-color: #e4e4e4;
         }
     }
 
-    :global(a.nav-link.active) {
+    :global(.active) .NavItem {
```

Comment:
This is a good change, but Ember currently recommends against using the global attribute except where necessary: https://ilyaradchenko.com/6-tips-for-transitioning-to-ember-css-modules/. Is there an opportunity here to remove this completely?

Reply:
The navigation tab is using [ember bootstrap](https://github.com/kaliber5/ember-bootstrap) and this library uses global classes to set styles. There may be a way to set these styles relying only on local classes, but that may be more work than needed given that ember bootstrap is setting these classes for us.

Reply:
Ok! I had not previously worked on a project that had a framework externalize CSS. Is there a reason Ember does this?

Reply:
And perhaps externalize is not the best word, but abstract.

Reply:
ember-css-modules is an external package, not a built-in ember feature.  But basically it helps to keep the css namespace clean and prevent bad class-names from messing up your styling among other things.

Reply:
got it, ok!